### PR TITLE
Create 2023-04-13-content.md

### PR DESCRIPTION
**Pull request by ptobia**
## Purpose of this pull request

This pull request (PR) will provide release notes for CSE content update of April 13, 2023

Issue number: N/A

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by kimsauce — COMMENTED** [review]
@ptobia double-checking 4/13 is the intended date?

**Comment by ptobia** [comment]
> @ptobia double-checking 4/13 is the intended date?

Yes I missed last week. I'll be following up shortly with this week's release.

File: blog-cse/2023-04-13-content.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* [New] Zoom - Group Modifications
+* [New] Zoom - Information Barrier Policy Modifications
+* [New] Zoom - Meeting Risk Alert
+* [New] Zoom - Recording Deleted or Trashed
+* [Updated] AWSGuardDuty_PenTest
+* [Updated] AWSGuardDuty_Stealth
+* [Updated] Recon_EC2_PortProbeUnprotectedPort
+* [Updated] Recon_IAMUser
+
+## Parsers
```
Comment (kimsauce) [inline blog-cse/2023-04-13-content.md:48]:
```suggestion
#### Parsers
```

File: blog-cse/2023-04-13-content.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* [New] MATCH-S00862 Zoom - Meeting Risk Alert
+* [New] MATCH-S00863 Zoom - Recording Modification
+* [Updated] THRESHOLD-S00096 Brute Force Attempt
+* [Updated] MATCH-S00565 Direct Outbound DNS Traffic
+* [Updated] THRESHOLD-S00103 Domain Brute Force Attempt
+* [Updated] THRESHOLD-S00102 Domain Password Attack
+* [Updated] THRESHOLD-S00095 Password Attack
+* [Updated] CHAIN-S00008 Successful Brute Force
+
+## Log Mappers
```
Comment (kimsauce) [inline blog-cse/2023-04-13-content.md:33]:
```suggestion
#### Log Mappers
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
H4 headers to match previous CSE release notes

**Comment by ptobia** [comment]
> H4 headers to match previous CSE release notes

Thanks Kim! Committed.

**Review by jpipkin1 — APPROVED** [review]
Ditto to @kimsauce's comments. Approved.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**